ArduCopter/arming_checks.cpp

```diff
@@ -425,19 +315,24 @@ bool Copter::pilot_throttle_checks(bool display_failure)
 }
 
 // perform pre_arm_rc_checks checks and set ap.pre_arm_rc_check flag
-void Copter::pre_arm_rc_checks()
+void AP_Arming_Copter::pre_arm_rc_checks()
 {
     // exit immediately if we've already successfully performed the pre-arm rc check
-    if (ap.pre_arm_rc_check) {
+    if (copter.ap.pre_arm_rc_check) {
         return;
     }
 
     // set rc-checks to success if RC checks are disabled
-    if ((g.arming_check != ARMING_CHECK_ALL) && !(g.arming_check & ARMING_CHECK_RC)) {
+    if ((checks_to_perform != ARMING_CHECK_ALL) && !(checks_to_perform & ARMING_CHECK_RC)) {
         set_pre_arm_rc_check(true);
         return;
     }
 
+    RC_Channel *&channel_roll = copter.channel_roll;
```

Comment:
Yes, these lines are intentional.  These lines reduce code churn and (more importantly) make the patch (much!) shorter and easier to understand.  Adding one line can save dozens of line in a diff.